### ISSUE TITLE
Support Go 1.9

### ISSUE THREAD

**Issue by moderation** [aporeto-inc/trireme-example#20]
I'm trying to build the Trireme example using Go 1.9 on Linux and the build is failing with the following error.

```
CGO_ENABLED=1 go build -a -installsuffix cgo
# github.com/aporeto-inc/trireme-example/vendor/github.com/aporeto-inc/trireme/enforcer
vendor/github.com/aporeto-inc/trireme/enforcer/nflog_linux.go:37:22: cannot assign 1 values to 2 variables
vendor/github.com/aporeto-inc/trireme/enforcer/nflog_linux.go:38:22: cannot assign 1 values to 2 variables
vendor/github.com/aporeto-inc/trireme/enforcer/nflog_linux.go:76:29: buf.Prefix undefined (type *nflog.NfPacket has no field or method Prefix)
vendor/github.com/aporeto-inc/trireme/enforcer/nflog_linux.go:79:89: buf.Prefix undefined (type *nflog.NfPacket has no field or method Prefix)
vendor/github.com/aporeto-inc/trireme/enforcer/nflog_linux.go:83:27: buf.Prefix undefined (type *nflog.NfPacket has no field or method Prefix)
Makefile:9: recipe for target 'build' failed
make: *** [build] Error 2
```

This is on a fresh `git clone` and `glide up`.

**Comment by satyamsi** [comment]
Instead of glide up, can you try glide install ? Its possible that the latest version of API is not ported yet.

Satyam

**Comment by moderation** [comment]
I just tried `glide install` and same error.

**Comment by satyamsi** [comment]
Did you do the "glide install" after reverting what "glide up" had done earlier ?

Just to be safe lets try the following commands:

git reset HEAD —hard
make build

I do see that the latest run on travis is passing:

https://travis-ci.org/aporeto-inc/trireme-example

Satyam

**Comment by bvandewalle** [comment]
Moved Travis to 1.9 in #33

**Comment by bvandewalle** [comment]
merged with #33 , closing
